helm_charts/obsrv-query-api/values.yaml

```diff
@@ -0,0 +1,8 @@
+namespace: "obsrv-query-engine"
+dockerhub: "gayathrisrividya"
```

Comment:
can we variablise?  let us create an env specific deployment configuration and use those values